spec/formatting.md

```diff
@@ -297,21 +297,28 @@ Next, using `res`, resolve the preferential order for all message keys:
 
 1. Let `pref` be a new empty list of lists of strings.
 1. For each index `i` in `res`:
-   1. Let `keys` be a new empty list of strings.
+   1. Let `keys` be a new empty set of strings.
```

Comment:
It was a list before on purpose: because the order of the keys matters when matching up to the selectors. The term "set" doesn't imply ordering and the term "list" does not require an implementation to use a list type or prevent the use of a (ordered) set type as long as other requirements are met. 

One could also say "array" here, but that is somewhat prejudicing the implementation?